kernels/histogram.cl

```diff
@@ -0,0 +1,40 @@
+// adapted code from https://github.com/bgaster/opencl-book-samples/blob/master/src/Chapter_14/histogram/histogram_image.cl
+
+#pragma OPENCL EXTENSION cl_khr_local_int32_base_atomics : enable
+
+const sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+kernel void histogram(
+    IMAGE_src_TYPE src,
+    IMAGE_dst_TYPE dst,
+    float minimum,
+    float maximum,
+    int step_size_x,
+    int step_size_y,
+    int step_size_z
```

Comment:
steps are always set to 1 in pyClesperanto, and step_size_y is not used in the [histogram_3d.cl](https://github.com/clEsperanto/clij-opencl-kernels/blob/9374863985160dd80854a3576e9cf863ee9952dd/kernels/histogram_3d_x.cl#L22-L36) version.
Raising the need to keep those arguments ?